notNeededPackages.json

```
@@ -3738,6 +3738,10 @@
             "libraryName": "rgb-hex",
             "asOfVersion": "3.0.0"
         },
+        "riderize__passport-strava-oauth2": {
+            "libraryName": "@riderize/passport-strava-oauth2",
+            "asOfVersion": "1.1.1"
```

Added on commit https://github.com/auth0-community/auth0-socketio-jwt/commit/72f3846fcf8b9c5e8d3d0850806464152292a02d which belongs to v1.0.9
```suggestion
            "asOfVersion": "1.0.9"
```